### PR TITLE
Added endpoint to download file resource to PropertyResource

### LINKED ISSUE #995: Add endpoint that serves files for BeanPropertyResource
To access for example geometry files of visualization, a special GET request with the BeanPropertyResource uuid should then return the file.

### PR DESCRIPTION
**Pull request by JAmmermann-DLR**
Also fix minor bug in Swagger Documentation of SEI and CA resources.

Closes #995

### CODE REVIEW

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/modelaccess/PropertyResource.java [hunk 1]
```diff
@@ -80,6 +88,63 @@ public Response getProperty(@PathParam("propertyUuid") @ApiParam(value = "Uuid o
 		}
 	}
 
+	/** **/
+	@GET
+	@Path("/{propertyUuid}/" + RESOURCE)
+	@Produces(MediaType.APPLICATION_OCTET_STREAM)
+	@ApiOperation(
+			produces = "application/json",
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/modelaccess/PropertyResource.java:96]:
```suggestion
			produces = "application/octet-stream",
```

File: de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/modelaccess/PropertyResourceTest.java [hunk 2]
```diff
@@ -127,5 +133,34 @@ public void testPropertyComposedPut() throws JAXBException {
...
+		assertNotFoundResponse(getTestRequestBuilder(ModelAccessResource.PROPERTY + "/unknown/" + PropertyResource.RESOURCE).get());
+		// A boolean property doesn't have a resource
+		assertBadRequestResponse(
+				getTestRequestBuilder(ModelAccessResource.PROPERTY + "/" + beanBool.getUuid().toString() + "/" + PropertyResource.RESOURCE).get(),
+				PropertyResource.PROPERTY_DOES_NOT_CONTAIN_A_SERVEABLE_RESOURCE);
+	}
+	
+	@Test
+	public void testResourceGet() throws CoreException {
+		// Initial the bean does not contain a valid resource
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/modelaccess/PropertyResourceTest.java:146]:
```suggestion
		// Initially the bean does not contain a valid resource
```

File: de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/modelaccess/PropertyResourceTest.java [hunk 3]
```diff
@@ -127,5 +133,34 @@ public void testPropertyComposedPut() throws JAXBException {
...
+		assertBadRequestResponse(
+				getTestRequestBuilder(ModelAccessResource.PROPERTY + "/" + beanResource.getUuid().toString() + "/" + PropertyResource.RESOURCE).get(),
+				PropertyResource.PROPERTY_DOES_NOT_CONTAIN_A_SERVEABLE_RESOURCE);
+		
+		// Now create a valid file
+		IFile file = ed.getResourceSet().getProject().getFile("file[2].xls");
+		file.create(new ByteArrayInputStream(TEST_STRING.getBytes()), IResource.FORCE, null);
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/modelaccess/PropertyResourceTest.java:153]: `.xls` implies a binary file, but you write a text string into it. Should we maybe test it on a real binary file, and then compare bytes?

**Review by andriikovalov-dlr — CHANGES_REQUESTED** [review]
Looks good, just a minor point

File: de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/modelaccess/PropertyResourceTest.java (hunk 3 above)
Comment (JAmmermann-DLR) [reply de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/modelaccess/PropertyResourceTest.java]: Thanks for the review, I hope I changed it accordingly

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Review by andriikovalov-dlr — APPROVED** [review]

**Pull request merged**